controllers/managedfusion_controller.go

```diff
@@ -432,6 +444,7 @@ func (r *ManagedFusionReconciler) reconcilePrometheus() error {
 
 		r.prometheus.Spec = desired.Spec
 		r.prometheus.Spec.ExternalLabels["clusterId"] = string(clusterVersion.Spec.ClusterID)
+		r.prometheus.Spec.RemoteWrite[0].BearerToken = r.remoteWriteConfig.BearerToken
```

Comment:
If there is no key, this will simply be set to `""` and remote write will stop working, Agent won't complain about it
It will be logged in the code if the key is not present in the validation section